app/models/hardware.rb

```diff
@@ -210,14 +210,6 @@ def m_memory(_parent, xmlNode, _deletes)
     self.memory_mb = xmlNode.attributes["memsize"]
   end
 
-  def m_bios(_parent, xmlNode, _deletes)
```

Comment:
@roliveri @jerryk55 I assume this was used for smartstate scans, does this look familiar to you?  Was it previously used but not anymore?

Reply:
@agrare if it was it was from before my time.  I don't ever remember seeing it used.